blip-tap.md

```diff
@@ -924,28 +924,56 @@ accept if favorable
 
 ##### Request For Quote (`tap_rfq`)
 
-When a receiver wishes to receive `N` units of TAP asset ID `asset_id`, a new
-p2p message `tap_rfq` is sent with the following structure:
+In this section, we will describe the message type a node uses to request a
+quote from a channel counterparty. When the requesting node wants to obtain
+`N` units of a TAP asset identified by `in_asset_id`/`in_asset_group_key` and
+divest with `M` units of another TAP asset identified by
+`out_asset_id`/`out_asset_group_key`, it sends a new p2p message called
+`tap_rfq` to the channel counterparty. If either `in_asset_id` or `out_asset_id`
+consists entirely of zeros, it represents BTC instead of a TAP asset.
+
+The `tap_rfq` message has the following structure:
 
 1. type: ?? (`tap_rfq`)
 2. data:
+     * [`1*byte`:`version`]
      * [`32*byte`:`rfq_id`]
-     * [`32*byte`:`asset_id`]
-     * [`BigSize`:`asset_amt`]
+     * [`8*byte`:`expiry`]
+     * [`BigSize`:`asset_max_amount`]
      * [`BigSize`:`suggested_rate_tick`]
+     * [`32*byte`:`in_asset_id`]
+     * [`33*byte`:`in_asset_group_key`]
+     * [`32*byte`:`out_asset_id`]
+     * [`33*byte`:`out_asset_group_key`]
 
 where:
 
+* `version` is the version of the quote request message; the current version is
+  `0`
 * `rfq_id` is a randomly generate 32-byte value to uniquely identify this RFQ
   request
-* `asset_id` is the asset ID of the asset the receiver intends to receive
-* `asset_amt` is the amount of units of said asset
+* `expiry` is the Unix timestamp in seconds after which the quote request and
+  suggested rate tick is no longer valid
+* `asset_max_amount` is the maximum asset amount that the counterparty is
+  expected to receive/divest
 * `suggested_rate_tick` is the internal unit used for asset conversions. A tick
   is 1/10000th of a currency unit. It gives us up to 4 decimal places of
   precision (0.0001 or 0.01% or 1 bps). As an example, if the BTC/USD rate was
   $61,234.95, then we multiply that by 10,000 to arrive at the `usd_rate_tick`:
   `$61,234.95 * 10000 = 612,349,500`.  To convert back to our normal rate, we
-  decide by `10,000` to arrive back at `$61,234.95`.
+  decide by `10,000` to arrive back at `$61,234.95`
```

Comment:
All the language related to a "tick" should be updated. We know use `assets_units_per_msat` in the code.